test/test_sparse_csr.py

```diff
@@ -1045,6 +1045,125 @@ def test_resize(self, device, dtype):
             self.assertEqual(a.shape, (new_shape[-2], new_shape[-1]))
             self.assertEqual(a._nnz(), 5)
 
+    @skipMeta
+    @dtypes(torch.float, torch.bool)
+    @all_sparse_compressed_layouts()
+    def test_resize_as_sparse_compressed(self, device, dtype, layout):
+
+        def _check_resize_b_as_a(b, a, should_resize=True):
```

Comment:
Remove `should_resize` argument as unused:
```suggestion
        def _check_resize_b_as_a(b, a):
```